app/graphql/mutations/rock_and_pebbles/delete_rock_pebble_relationship.rb

```diff
@@ -0,0 +1,25 @@
+module Mutations
+  module RockAndPebbles
+    class DeleteRockPebbleRelationship < ::Mutations::BaseMutation
```

Comment:
I've got a question about the name of this, not sure what the answer is; you named it after the effect of the mutation on the database, which makes sense, but should we name it after the way we intend it to be used instead? I.e., "DeclineRockPebbleRelationship"? I'm not sure whether there's a GraphQL convention around this or not. I'm slightly inclined toward renaming because of the email side-effect but could go either way.

Reply:
I went back and forth with this as well. I'm open to suggestions/consensus.

Reply:
Rename this to decline_rock_pebble_relationship